packages/clay-badge/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+const HWP = require('html-webpack-plugin');
+
+module.exports = {
+	entry: path.join(__dirname, './demo/App.tsx'),
```

Comment:
nit: if you're using `path.join`, you don't need the "./" in "./demo/App.tsx".

Reply:
Updated this in the generator